nni/compression/pytorch/compressor.py

```diff
@@ -511,7 +511,7 @@ def __init__(self, model, config_list, optimizer=None):
                     # and it is trainable, therefore, it should be added to optimizer.
                     self.optimizer.add_param_group({"params": wrapper.module.old_weight})
 
-    def quantize_weight(self, weight, wrapper, **kwargs):
+    def quantize_weight(self, wrapper, **kwargs):
```

Comment:
why remove `weight`? it would be better to keep it for now

Reply:
All derived classes of Quantizer have deleted `weight` in function `quantize_weight`. It should be deleted in preview pr but be kept until now,